jc/parsers/efibootmgr.py

```diff
@@ -0,0 +1,144 @@
+"""jc - JSON Convert `efibootmgr` command output parser
+
+Only `efibootmgr -v` option is supported with the default output.
+
+Usage (cli):
+
+    $ sudo efibootmgr | jc --efibootmgr
+    $ sudo efibootmgr -v | jc --efibootmgr
+
+or
+
+    $ sudo jc efibootmgr
+
+Usage (module):
+
+    import jc
+    result = jc.parse('efibootmgr', efibootmgr_command_output)
+
+Schema:
+
+[
+    {
+        "BootCurrent": string,
+        "Timeout": string,
+        "BootOrder": [
+            string,
+            string,
+            string
+        ],
+        "BootOptions": [
+            {
+                "BootOptionReference": string,
+                "DisplayName": string,
+                "UefiDevicePath": string,
+                "BootOptionEnabled": bool
+            },
+            {
+                "BootOptionReference": string,
+                "DisplayName": string,
+                "UefiDevicePath": string,
+                "BootOptionEnabled": bool
+            },
+            {
+                "BootOptionReference": string,
+                "DisplayName": string,
+                "UefiDevicePath": string,
+                "BootOptionEnabled": bool
+            }
+        ]
+    }
+]
+
+Examples:
+
+$ sudo efibootmgr -v | jc --efibootmgr --pretty
+[
+  {
+    "BootCurrent": "0002",
+    "Timeout": "0 seconds",
```

Comment:
I would rename this field `timeout_seconds` and convert it to an integer. (unless it can be a float, then convert to float). This can be done in the `_process()` function.

Reply:
From `efibootmgr` help manual the unit is seconds.